tests/system/src/test/java/test/robot/helloworld/CustomSecurityManagerTest.java

```diff
@@ -201,11 +201,11 @@ private void doTestFullScreenCommon(SecurityManager sm,
                     // avoid the top area as it might contain OS-specific UI (Macs with a notch)
                     y = h / 3;
```

Comment:
Now that this tests uses visualBounds, there is no need to worry about avoiding the top, so this can be something like "y = 4".

Reply:
When the app enters full screen mode in MacBook Pro M1, there will be still a black strip on the top because of the notch. So assigning small value will not fetch green color from app window. It requires at least y=40.
So I think we can add new static final variable with a value of 40 or 50 to be on the safer side and use it in all the places so that it works for all the cases.
Please let me know your thought on this.

Reply:
Updated the code to use visual bounds while calculating `x` and `y` values.
Changed the offset of coordinates used for reading color to 10 since in M1 macs `robot` reads color value from the cursor position irrespective of  coordinates given and cursor is parked at the bottom right corner with an offset of 5 from the visual bounds. Hence full screen tests where failing if `getMaxY()` method was used to calculate y value with lower offsets.